mobile/src/Navigator/index.js

```diff
@@ -28,6 +28,9 @@ const AppContainer = () => {
                 teamAvatar,
                 saveTeamAvatar,
                 clearStorage,
+                handleSubscribeToGame,
+                handleAddTeam,
+                handleAddTeamAnswer
```

Comment:
hoisted these functions from individual components to container so we don't need to duplicate API code. There might be a rerender cost to this but I feel like it's cleaner